### PR TITLE
DOCSP-32186 Version Information

### PR DESCRIPTION
**Pull request by kennethdyer**
## Description

Clarifies supported version for mongosync.

## Staging

[Currently Supported Versions](https://preview-mongodbkennethdyer.gatsbyjs.io/cluster-sync/DOCSP-32186-c2c-version/reference/versioning/#currently-supported-versions)

## JIRA

[DOCSP-32186](https://jira.mongodb.org/browse/DOCSP-32186)

## Build

* [2024-02-27](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=65de5da57111165982e80429)
* [2024-03-01](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=65e227f311db7b04279ef2b1)

### CODE REVIEW

File: source/reference/versioning.txt [hunk 1]
```diff
@@ -27,12 +27,14 @@ version number for {+c2c-product-name+}:
...
-
 Currently Supported Versions
 ----------------------------
 
-If you are using an unsupported version of ``mongosync``, you may be
-asked to upgrade in order to receive support.
+MongoDB only supports the latest patch release of each major
+version.  For example, ``mongosync`` {+latest-version+} is the
+latest patch release of {+version+}. If you are using an
+unsupported version of ``mongosync``, you may be asked to
```
Comment (jmd-mongo) [inline source/reference/versioning.txt:36]: [nit]

This ought to be "might" on line 36 according to the style guide: https://www.mongodb.com/docs/meta/style-guide/writing/use-effective-verbs/#std-label-helping-verbs

**Review by jmd-mongo — APPROVED** [review]
Thanks for this update, @kennethdyer! This one LGTM mod a tiny nit.

Thank you,
Joe

File: source/reference/versioning.txt (hunk 1 above)
Comment (kennethdyer) [reply source/reference/versioning.txt]: Fixed

**Review by autarch — APPROVED** [review]
LGTM!

**Pull request merged**